src/storage/chaindb.md

```diff
@@ -0,0 +1,84 @@
+# The ChainDB format
+
+The format in which the Cardano blockchain is stored in the disk has been historically the ChainDB format used by IOG's `cardano-node`. It consists of 3 separate directories in which different data is stored.
```

Comment:
> The format in which the Cardano blockchain is stored in the disk has been historically the ChainDB format used by IOG's `cardano-node`.

This information was already given in src/storage/README.md In any case we should change this to "the reference Cardano implementation" or something like that.

Reply:
I moved some parts of that other apge to this one